test/test_operations.py

```diff
@@ -621,13 +621,36 @@ def test_index_put(self):
     vset = b.sum().item()
     self.assertEqual(a.sum().item(), 10.0 * vset + (4.0 - vset))
 
-  def test_pow_integer_types(self):
+  def test_fall_back_integer_types(self):
```

Comment:
Please break down these on a per operation tests.
Easier to understand what failed from the test name.